webapp/components/create_post.jsx

```diff
@@ -320,6 +323,24 @@ class CreatePost extends React.Component {
     componentWillUnmount() {
         ChannelStore.removeChangeListener(this.onChange);
         PreferenceStore.removeChangeListener(this.onPreferenceChange);
+        document.removeEventListener('keydown', this.showShortcuts);
+    }
+    showShortcuts(e) {
+        if ((e.ctrlKey || e.metaKey) && e.keyCode === Constants.KeyCodes.FORWARD_SLASH) {
```

Comment:
Yes, but it does not affect what you are typing, you do not lose anything by pressing this.